### ISSUE TITLE
Consider changing versioning strategy

### ISSUE THREAD

**Issue by paulmillr** [ethereumjs/ethereumjs-monorepo#3748]
All packages currently have different versioning scheme. Here's a couple thoughts how can this be improved:

a. Adopt one double-digit version for all packages e.g. v10.
b. Use year as version. "2024" can be shortened to just v24. That would automatically make v25 breaking. Makes easier to understand which hardfork the particular version supports just by its year

Benefits of using the same number:

1. Easier bug tracking / user reporting / dev UX. E.g. "We're on version 11, this happens" instead of "We're on version 4.4.0 with vm 0.10.2, this happens".
2. Makes it easier to continue developing monorepo and reason about changes

### Current package versions

```
"5.3.0"
"7.3.0"
"0.10.2"
"4.4.0"
"6.1.3"
"3.0.4"
"3.1.0"
"0.2.3"
"6.2.2"
"5.0.2"
"2.4.0"
"5.4.0"
"9.1.0"
"0.1.0"
"8.1.0"
"2.0.4"
```

**Comment by holgerd77** [comment]
Yeah, I think that makes some sense, we'll give this some thought.

I see the (additional) benefits of this year-version scheme as well, my preference - in case we decide to do - would be the double-digit alignment though I guess.

We sometimes do breaking "in-between" releases for a subset of (the higher-level) libraries, e.g. only EVM/VM lately, and then we could still do that and e.g. go from v11 to v12 only for these libraries (and still confuse people a bit, but so be it, still a lot better to grasp and communicate than all versions being on a different level).

Also we are then not fully bound to this one-breaking-release-a-year scheme, *somewhat* works for us but I am not sure if I would feel fully comfortable with the sight of not being able to do breaking releases if necessary just because the versioning scheme doesn't fit.

**Comment by holgerd77** [comment]
Just to note here: we have decided to give with the double digit numbering suggested in a) starting with the upcoming breaking releases, thanks, great suggestion! 🙏

**Comment by paulmillr** [comment]
@holgerd77 is the plan to bump version in alpha-2? The last release still indicates [old scheme](https://github.com/ethereumjs/ethereumjs-monorepo/commit/c77fb81a72d71bd6af1fbb97688241018d1beb55)

**Comment by acolytec3** [comment]
Based on internal discussions, sounds like we're all agreed that we will move to a single version number for the next releases, that are coming soon :TM:
